Animations/Fade/DownBig.js

```diff
@@ -26,5 +28,5 @@ var FadeDownBig = {
     'transform': 'translateY(-2000px)'
   }
 };
-//Use makeTransition function from main Tux Animation module to create wrapping animation componenet and pass in the default params
-module.exports = Animation.makeTransition('FadeDownBig', FadeDownBig);
+//Use makeAnimation function from main Tux Animation module to create wrapping animation componenet and pass in the default params
+module.exports = makeAnimation('FadeDownBig', FadeDownBig);
```

Comment:
This does not need to have `'FadeDownBig'` passed into it since we have defined a `className` property in `FadeDownBig`.